requirements.txt

```diff
@@ -30,6 +30,7 @@ psutil==5.9.2
 pyarrow==9.0.0
 pyerfa==2.0.0.1
 pyparsing==3.0.9
+pytest
```

Comment:
as a note: according to the [LSST DM Guide](https://developer.lsst.io/python/testing.html) while all LSST stuff is supposed to be able to be run by pytest, the tests are supposed to be written in the `unittest` framework (pytest is backwards compatible with unittest but not vice versa). It's not the biggest issue but it might be nice to be consistent with other Rubin projects 😊

Reply:
There is a dask test instance that is pytest-only (I haven't yet gotten it to work, but it exists). I had migrated tests over to use pytest while playing around with it, and found out how much nicer pytest is than unittest!

Based on your comments below, you agree that pytest is more fun, and we should just stick to it since we have the opportunity to use pytest from the start.